src/book/services/BooksService.test.tsx

```diff
@@ -79,8 +79,12 @@ describe("BookService", () => {
     // given
     expect.hasAssertions();
     const bookToCheck = mockedResponseBooks[0];
+
+    const TestWrapper = ({ children }: { children: any }) => {
+      return <LoaderProvider>{children}</LoaderProvider>;
+    };
     // when
-    const { result } = renderHook(() => useBooks());
+    const { result } = renderHook(() => useBooks(), { wrapper: TestWrapper });
```

Comment:
here useBook hook calls inside another hook, when we test with renderHook, we need to use a wrapper